bpython/curtsiesfrontend/repl.py

```diff
@@ -994,15 +996,58 @@ def send_current_block_to_external_editor(self, filename=None):
         self.cursor_offset = len(self.current_line)
 
     def send_session_to_external_editor(self, filename=None):
+        """
+        Sends entire bpython session to external editor to be edited. Usually called with F7. 
```

Comment:
```suggestion
        Sends entire bpython session to external editor to be edited. Usually bound to F7. 
```
to me "called with" means the arguments passed in